src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrPatchHandler.cs

```diff
@@ -529,7 +540,11 @@ public IReadOnlyCollection<string> GetVmrPatches(SourceMapping mapping)
             return Array.Empty<string>();
         }
 
-        var mappingPatchesPath = _fileSystem.PathCombine(_vmrInfo.PatchesPath, mapping.Name);
+        var mappingPatchesPath = _fileSystem.PathCombine(
+            _vmrInfo.VmrPath,
+            _vmrInfo.PatchesPath.Replace('/', Path.DirectorySeparatorChar),
```

Comment:
Above you utilized `_fileSystem.DirectorySeparatorChar`, while here you chose `Path.DirectorySeparatorChar`. Is there some difference between the two?

Reply:
Should be the `_fileSystem` one, thanks!